trials/mainnightly/services/init-service-dev.yml

```diff
@@ -52,6 +52,15 @@ mts:
       - name: CCO
         alias: CCO
         siteType: CCO
+        address:
+          address1: address1
```

Comment:
as per the site validation shouldnt it be - 
- name : addressLine1  etc ?

Reply:
no validation is yet covered @darrengsmithson just the entities are covered as part of arts-207, in the future we might be covering it :)

Reply:
ok but we should at least try to keep the site-service-dev.yml and init-service-dev.yml in sync to avoid confusion..I think?

Reply:
Done it